subsystem/authz_test.go

```diff
@@ -203,9 +203,9 @@ var _ = Describe("Make sure that sensitive files are accessible only by owners o
 		})
 		it = fmt.Sprintf("Should not allow read-only-admins  to download '%v' via downloads/files-presigned endpoint", fileName)
 		It(it, func() {
-			_, err := readOnlyAdminUserBMClient.Installer.GetPresignedForClusterFiles(ctx, &installer.GetPresignedForClusterFilesParams{ClusterID: clusterID, FileName: fileName})
+			_, err := readOnlyAdminUserBMClient.Installer.V2GetPresignedForClusterFiles(ctx, &installer.V2GetPresignedForClusterFilesParams{ClusterID: clusterID, FileName: fileName})
 			Expect(err).To(HaveOccurred())
-			Expect(reflect.TypeOf(err)).Should(Equal(reflect.TypeOf(installer.NewGetPresignedForClusterFilesForbidden())))
+			Expect(reflect.TypeOf(err)).Should(Equal(reflect.TypeOf(installer.NewV2GetPresignedForClusterFilesForbidden())))
```

Comment:
While we're here can you make this assertion a bit more helpful?
This is not a good failure message:
```
/go/src/github.com/openshift/assisted-service/subsystem/authz_test.go:205
Expected
    <*reflect.rtype | 0x1a8efc0>: {size: 0x8, ptrdata: 0x8, hash: 3563199731, tflag: 9, align: 8, fieldAlign: 8, kind: 54, equal: 0x404700, gcdata: 1, str: 157180, ptrToThis: 0}
to equal
    <*reflect.rtype | 0x1acf4a0>: {size: 0x8, ptrdata: 0x8, hash: 2194802269, tflag: 9, align: 8, fieldAlign: 8, kind: 54, equal: 0x404700, gcdata: 1, str: 608164, ptrToThis: 0}
/go/src/github.com/openshift/assisted-service/subsystem/authz_test.go:208
```

Reply:
There's a gomega built-in matcher `BeAssignableToTypeOf` that I think would work here.